services/traefik/docker-compose.yml.j2

```diff
@@ -137,6 +143,12 @@ services:
         - traefik.http.middlewares.authenticated_platform_user.forwardauth.address=http://${WEBSERVER_HOST}:${WEBSERVER_PORT}/v0/auth:check
         - traefik.http.middlewares.authenticated_platform_user.forwardauth.trustForwardHeader=true
         - traefik.http.middlewares.authenticated_platform_user.forwardauth.authResponseHeaders=Set-Cookie,osparc-sc2
+        #
+        # middleware: http(s)://(www.) to  https://
+        - traefik.http.middlewares.strip-www.redirectregex.regex=^(.*)www\.(.+)
```

Comment:
I guess this will also capture and transform the URL below. Shall we use stricter regex?

`https://domain.com/www.news.html`

Note: dots in URL path are allowed according to this [SO thread](https://stackoverflow.com/questions/6777274/can-urls-contain-dots-in-the-path-part)

Reply:
I enhanced the regex to take care of this and be stricter :--)